royalties/contracts/impl/AbstractRoyalties.sol

```diff
@@ -5,25 +5,28 @@ pragma solidity >=0.6.2 <0.8.0;
 import "../LibPart.sol";
 
 abstract contract AbstractRoyalties {
-    mapping (uint256 => LibPart.Part[]) public royalties;
+    mapping (uint256 => LibPart.Part[]) internal _royalties;
```

Comment:
I think that's not possible to change names for fields. there will be an error if I try to upgrade the contract

Reply:
I've reverted this storage variable name.  Kept the visibility internal (as there is no reason for it to be public).